packages/patternfly-4/src/components/_react/Documentation/SideNav.js

```diff
@@ -125,6 +139,22 @@ class SideNav extends React.Component {
                   </NavItem>
                 ))}
               </NavExpandable>
+              <NavExpandable
+                title="Charts"
+                isExpanded={currentPath.indexOf('/charts/') > -1 || Boolean(searchValue && filteredLayoutRoutes.length > 0)}
```

Comment:
I guess we removed sidebar search but probably still good to fix or remove all remnants of that search
```suggestion
                isExpanded={currentPath.indexOf('/charts/') > -1 || Boolean(searchValue && filteredChartRoutes.length > 0)}
```